features/minion/core/features/src/main/resources/features.xml

```diff
@@ -1,6 +1,9 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <features name="minion-core-features" xmlns="http://karaf.apache.org/xmlns/features/v1.2.0">
 
+    <!-- OpenNMS Features -->
+    <repository>mvn:org.opennms.karaf/opennms/${project.version}/xml/features</repository>
```

Comment:
We don't want to import these dependencies with the "core" features, let's create a new project for the provisioning shell commands add with the "default" features instead.